components/studentPortal/layout/Sidebar.tsx

```diff
@@ -192,7 +192,7 @@ export const Sidebar: React.FC = () => {
         {userRole === "coach" && (
           <SidebarItem
             name={"Admin"}
-            link={"/studentPortal/admin"}
+            link={"/profile/admin"}
```

Comment:
From a code organization perspective, I think the admin page should be part of the student portal not the profile